src/clients/c++/examples/shm_utils.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2018-2019, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Are the utils here similar to the one we have for Python shared memory library? If so, we can probably keep one instead of having duplicates.

Reply:
Yes these are similar to the ones used in the Python client but both do slightly different things. The python side implements one function each for creation, setting and deletion and uses the shm handle. These functions are in some way the building blocks of the python shared memory client library